test/e2e/network/loadbalancer.go

```diff
@@ -987,6 +1061,14 @@ var _ = common.SIGDescribe("LoadBalancers ExternalTrafficPolicy: Local", feature
 			framework.ExpectNoError(err)
 		})
 
+		// FIXME: figure out the actual expected semantics for
+		// "ExternalTrafficPolicy: Local" + "IPMode: Proxy".
+		// https://issues.k8s.io/123714
+		ingress := &svc.Status.LoadBalancer.Ingress[0]
+		if ingress.IP == "" || (ingress.IPMode != nil && *ingress.IPMode == v1.LoadBalancerIPModeProxy) {
+			e2eskipper.Skipf("LoadBalancer uses 'Proxy' IPMode")
```

Comment:
this is working fine, I can see how these tests are skipped now https://testgrid.k8s.io/sig-network-kind#pr-sig-network-kind,%20loadbalancer